File: app/src/main/java/com/github/doomsdayrs/apps/shosetsu/ui/reader/demarkActions/ParaSpacingChange.java
package com.github.doomsdayrs.apps.shosetsu.ui.reader.demarkActions;

import com.github.doomsdayrs.apps.shosetsu.backend.Utilities;
import com.github.doomsdayrs.apps.shosetsu.ui.reader.NewChapterReader;

import static com.github.doomsdayrs.apps.shosetsu.backend.Utilities.changeParagraphSpacing;

public class ParaSpacingChange implements Utilities.DemarkAction {
    private final NewChapterReader chapterReader;

    public ParaSpacingChange(NewChapterReader chapterReader) {
        this.chapterReader = chapterReader;
    }

    @Override
    public void action(int spared) {
        changeParagraphSpacing(spared);
        chapterReader.currentView.setUpReader();
    }

}

File: app/src/main/java/com/github/doomsdayrs/apps/shosetsu/ui/reader/demarkActions/ParaSpacingChange.kt
package com.github.doomsdayrs.apps.shosetsu.ui.reader.demarkActions

import com.github.doomsdayrs.apps.shosetsu.backend.DeMarkAction
import com.github.doomsdayrs.apps.shosetsu.backend.Settings
import com.github.doomsdayrs.apps.shosetsu.ui.reader.ChapterReader

class ParaSpacingChange(private val chapterReader: ChapterReader) : DeMarkAction {
	override fun action(spared: Int) {
		Settings.readerParagraphSpacing = (spared)
//		chapterReader.setUpReader()
	}
}